content/millennium/r4/other/binary.md

```diff
@@ -61,6 +62,34 @@ The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentR
 
 <%= disclaimer %>
 
+### Example
```

Comment:
Looks like we are having to endup creating a lot of examples to help guide consumers, how about we do something similar to what was done in DSTU2?
We could call out that the headers that can be passed in are as defined here: https://fhir.cerner.com/millennium/dstu2/infrastructure/binary/#headers

And we could provide one example of 'FHIR Accept Type' and an other 'Native Accept Type'. Thoughts?

Reply:
Updated here: 5da58d896172434cc5898acbdeb87f769ee90bd9